src/node_contextify.cc

```diff
@@ -371,10 +401,26 @@ ContextifyContext* ContextifyContext::ContextFromContextifiedSandbox(
 // static
 template <typename T>
 ContextifyContext* ContextifyContext::Get(const PropertyCallbackInfo<T>& args) {
-  Local<Value> data = args.Data();
+  Local<Context> context;
+  if (!args.This()->GetCreationContext().ToLocal(&context)) {
+    return nullptr;
+  }
+  if (UNLIKELY(context->GetNumberOfEmbedderDataFields() <=
+               ContextEmbedderIndex::kContextifyContext)) {
+    return nullptr;
+  }
+  if (UNLIKELY(context->GetAlignedPointerFromEmbedderData(
+                   ContextEmbedderIndex::kContextTag) !=
+               Environment::kNodeContextTagPtr)) {
```

Comment:
Do you want to open a separate PR for that? It would be better to rebase this one on top of that, instead of the entire https://github.com/nodejs/node/pull/44179

Reply:
Yeah, I'll submit a separate PR for it.